packages/farm-ui/package.json

```diff
@@ -1,16 +1,26 @@
 {
   "name": "farm-ui",
   "version": "2.0.0-alpha.1",
-  "description": "",
+  "type": "module",
```

Comment:
Just making a reminder for myself to make sure this works w/ the field-kit module.